rust_snuba/src/factory.rs

```diff
@@ -116,6 +116,7 @@ impl ProcessingStrategyFactory<KafkaPayload> for ConsumerStrategyFactory {
             &self.clickhouse_concurrency,
             &self.storage_config.clickhouse_cluster.user,
             &self.storage_config.clickhouse_cluster.password,
+            self.storage_config.clickhouse_cluster.async_inserts,
```

Comment:
I was thinking for a minute this would actually be easier to have as a clickhouse config but it seems like a much more involved change than I thought so going to move this to `cli` arg